library.gradle

```diff
@@ -7,6 +7,16 @@ android {
     compileSdkVersion compileVersion
     buildToolsVersion buildToolsVersion
 
+    flavorDimensions "extraAPIs"
+    productFlavors {
+        defaults {
+            dimension "extraAPIs"
+        }
+        integrationTest {
+            dimension "extraAPIs"
+        }
+    }
```

Comment:
Note that I added these flavors for all modules instead of just the `purchases` module (the one that needs to have additional APIs). This simplifies developer life in Android studio I think.

Reply:
Hmm seems unnecessary considering one of the flavors is already named `integrationTest`? The dimension should describe what the difference between the flavors will be. While `extraAPIsForIntegrationTests` gives more detail on what this is, the exact difference is that one of the flavors will have extra APIs... NABD though, I could go either way.

Reply:
Going to merge without this change, but lmk if you prefer it and I can change it in another PR.